src/dotnet/commands/dotnet-migrate/Program.cs

```diff
@@ -42,7 +42,8 @@ public static int Run(string[] args)
             CommandOption skipProjectReferences = app.Option("-s|--skip-project-references", "Skip migrating project references. By default project references are migrated recursively", CommandOptionType.BoolValue);
             
             CommandOption reportFile = app.Option("-r|--report-file", "Output migration report to a file in addition to the console.", CommandOptionType.SingleValue);
-            CommandOption structuredReportOutput = app.Option("--format-report-file-json", "Output migration report file as json rather than user messages", CommandOptionType.BoolValue); 
+            CommandOption structuredReportOutput = app.Option("--format-report-file-json", "Output migration report file as json rather than user messages", CommandOptionType.BoolValue);
+            CommandOption doBackup = app.Option("--do-backup", "Move project.json, global.json, and *.xproj to a `.backup` directory after successful migration.", CommandOptionType.BoolValue); 
```

Comment:
I think this should be called simply backup. Same for the variable in the class above.